content/docs/2.6/scalers/predictkube.md

```diff
@@ -0,0 +1,102 @@
++++
+title = "Predictkube"
+availability = "v2.6+"
+maintainer = "Dysnix"
+description = "AI-based predictive scaling based on Prometheus metrics & PredictKube SaaS."
+layout = "scaler"
+go_file = "predictkube_scaler"
++++
+
+PredictKube is an open-source project that uses a SaaS offering consisting of an AI model consumed by connecting to a REST API for the main functions of predicting and scaling.
+
+To make our AI model access your data and make a prediction based on it, please use this API key we provide in PredictKube.
+
+Review our [Privacy Policy](https://predictkube.com/privacy-policy) to see how your data circulates in and out PredictKube.
+
+### Trigger Specification
+
+This specification describes the `predictkube` trigger that scales based on a predicting load based on `prometheus` metrics.
+
+```yaml
+triggers:
+- type: predictkube
+  metadata:
+    # Required fields:
+    predictHorizon: "2h"
+    historyTimeWindow: "7d"
+    prometheusAddress: http://<prometheus-host>:9090
+    metricName: http_requests_total
+    query: sum(rate(http_requests_total{deployment="my-deployment"}[2m]))
+    queryStep: "2m"
+    threshold: '100'
+```
+
+**Parameter list:**
+
+- `predictHorizon` - Prediction time interval. It is usually equal to the cool-down period of your application.
+- `historyTimeWindow` - Time range for which to request metrics from Prometheus. We recommend using minimum 7-14 days time window as historical data.
+- `prometheusAddress` - Address of Prometheus server.
+- `metricName` - Name to identify the Metric in the external.metrics.k8s.io API. API. If using more than one trigger it is required that all `metricName`(s) be unique.
```

Comment:
As mentioned on the main PR, we are trying to not expose this property for users, unless there is a really good reason.